tests/test_patchy.py

```diff
@@ -39,6 +39,13 @@ def sample():
             """)
         assert sample() == 2
 
+    def test_replace(self):
```

Comment:
Looking more closely, you've added this test to the `TestPatch` class when really it should have been added to a `TestReplace` class - there's one test class for each function in the public API. I'm changing this now whilst adding more tests.